app/controllers/api/v1/users_controller.rb

```diff
@@ -1,9 +1,19 @@
 class Api::V1::UsersController < ApiController
 
+
+  api :GET, '/v1/users/','List of resources: User'
+  formats ['json']
```

Comment:
The routes file says that the formats for this route are `format: /(text|json)/`. This one only has `json`. Which one do we want to stick with? Same with the show action.